examples/src/main/java/dataset.tsv

```diff
@@ -0,0 +1,203 @@
+url	label
```

Comment:
please move dataset.tsv to `src/main/resources` folder

Reply:
Done